### PR TITLE
chore(pipeline): create an action in the pipeline for local jobs

### PR DESCRIPTION
**Pull request by efekarakus**
_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: internal/pkg/workspace/workspace.go [hunk 1]
```diff
@@ -143,6 +143,13 @@ func (ws *Workspace) JobNames() ([]string, error) {
 	})
 }
 
+// WorkloadNames returns the name of all the workloads in the workspace.
+func (ws *Workspace) WorkloadNames() ([]string, error) {
```
Comment (iamhopaul123) [inline internal/pkg/workspace/workspace.go:147]: slick

**Comment by efekarakus** [comment]
Tested it and the pipeline creates now stages with local jobs and it deploys to it

**Review by iamhopaul123 — APPROVED** [review]
:shipit: Does pipeline work as expected for both service and job after this PR?

**Review by kohidave — APPROVED** [review]

File: internal/pkg/workspace/workspace.go [hunk 2]
```diff
@@ -143,6 +143,13 @@ func (ws *Workspace) JobNames() ([]string, error) {
...
+	return ws.workloadNames(func(wlType string) bool {
```
Comment (kohidave) [inline internal/pkg/workspace/workspace.go:148]: magic

**Review by kohidave — APPROVED** [review]

**Pull request merged**